lbrynet/lbrynet_daemon/DaemonControl.py

```diff
@@ -96,24 +126,34 @@ def start():
     if not args.logtoconsole and not args.quiet:
         print "Starting lbrynet-daemon from command line"
         print "To view activity, view the log file here: " + lbrynet_log
-        print "Web UI is available at http://%s:%i" % (API_INTERFACE, API_PORT)
-        print "JSONRPC API is available at " + API_CONNECTION_STRING
+        print "Web UI is available at http://%s:%i" % (settings.API_INTERFACE, settings.api_port)
+        print "JSONRPC API is available at " + settings.API_CONNECTION_STRING
         print "To quit press ctrl-c or call 'stop' via the API"
 
     if test_internet_connection():
         lbry = DaemonServer()
 
-        d = lbry.start(branch=args.branch if args.branch else DEFAULT_UI_BRANCH,
-                       user_specified=args.ui,
-                       wallet=args.wallet,
-                       branch_specified=True if args.branch else False)
+        d = lbry.start()
```

Comment:
Moving from arguments to a global configuration object is an anti-pattern. I accept that there will be some global configuration because that is how things were before this pull request, but we shouldn't be adding more global state.